lib/server/server-middleware.js

```diff
@@ -117,8 +118,13 @@ disallow: /public`
     allowDots: true,
     parseArrays: false
   }))
+  middleware.push(device.capture({parseUserAgent: true}))
```

Comment:
I can't see how we use this user agent.  Is it required to capture this?

Reply:
Ah - that was me just playing around. Anna and Mark (the content designer who worked on Save and Return) wanted to show the device the user logged in with (and where from).

Reply:
Removed